interop-tests/Cargo.toml

```diff
@@ -14,16 +14,31 @@ either = "1.9.0"
 env_logger = "0.10.0"
 futures = "0.3.28"
 log = "0.4"
-serde = { version = "1", features = ["derive"] }
 rand = "0.8.5"
+serde = { version = "1", features = ["derive"] }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 axum = "0.6"
-libp2p = { path = "../libp2p", features = ["ping", "noise", "tls", "rsa", "macros", "websocket", "tokio", "yamux", "tcp", "dns", "identify", "quic"] }
-libp2p-webrtc = { workspace = true, features = ["tokio"] }
+libp2p = { path = "../libp2p", features = [
+    "dns",
+    "identify",
+    "macros",
+    "noise",
+    "ping",
+    "quic",
+    "rsa",
+    "tcp",
+    "tls",
+    "tokio",
+    "websocket",
+    "yamux",
+] }
```

Comment:
We don't multiline these anywhere else. Can you please revert.

Reply:
Yes I can.